static/js/popovers.js

```diff
@@ -209,9 +209,9 @@ exports.toggle_actions_popover = function (element, id) {
             narrowed: narrow.active(),
         };
 
-        var ypos = elt.offset().top - message_viewport.scrollTop();
+        var ypos = elt.offset().top;
```

Comment:
Are you sure about removing the `scrollTop` subtraction?  I think that's important if there's a lot of history to get the height from the top of the viewport, not the height from the top of the feed (where there may be 1000+ messages)